provisioning/miniprov/hare_mp/store.py

```diff
@@ -92,8 +92,10 @@ def get_machine_ids_for_service(self, service_type: str) -> List[str]:
         service_type will be like Const.SERVICE_MOTR_IO.value,
         Const.SERVICE_S3_SERVER.value etc
         """
+        if service_type == 'io':
+            return self.get_machine_ids_for_attribute('num_cvg')
         types = {
-            'io': 'motr',
+            'motr_client': 'motr',
```

Comment:
Won't this give both data and client nodes then for service type `motr_client`?

Reply:
Yes, it will give both data and client nodes information, checking with Provisioning team